emails/views.py

```diff
@@ -513,15 +513,19 @@ def _sns_message(message_json):
     message_body = {}
     tracker_report_link = ""
     removed_count = 0
+    datetime_now = int(datetime.now(timezone.utc).timestamp() * 1000)
+
     if html_content:
         incr_if_enabled("email_with_html_content", 1)
         tracker_removal_flag = Flag.objects.filter(name="tracker_removal").first()
-        tracker_removal_flag_active = (
-            tracker_removal_flag
-            and tracker_removal_flag.is_active_for_user(address.user)
+        tracker_removal_flag_active = tracker_removal_flag and (
+            tracker_removal_flag.is_active_for_user(address.user)
+            or tracker_removal_flag.everyone
```

Comment:
👍 this is a gotcha with waffle flags. `flag.is_active_for_user` doesn't take `flag.everyone` into account, only if a flag was enabled for a specific user. I'm not sure why this works in production.